Quick_Deploy/vLLM/model_repository/vllm/config.pbtxt

```diff
@@ -0,0 +1,52 @@
+name: "vllm"
+backend: "python"
+max_batch_size: 0
+# We need to use decoupled transaction policy for saturating
+# vLLM engine for max throughtput.
+# TODO [DLIS:5233]: Allow asychronous execution to lift this
+# restriction for cases there is exactly a single response to
+# a single request.
+model_transaction_policy {
+  decoupled: True
+}
+
+# VLLM_ENGINE_ARGS_JSON parameters is serialized
+# json with VLLMAsyncEngineConfig settings. Look
+# at VLLMAsyncEngineConfig in model.py for supported
+# settings.
+parameters {
+  key: "VLLM_ENGINE_ARGS_JSON"
+  value: {
+    string_value: "{\"model\":\"facebook/opt-125m\"}"
```

Comment:
escaped json seems like it will be a bit hard to get right.

perhaps a `vllm_engine_args.json` file the model directly that you consume via the new `pb_utils.model_path()`.

and perhaps we could override that filename with the VLLM_ENGINE_ARGS_JSON parameter?

to me, a clean and ordinary .json file is easier to edit, than a single escaped string.

Reply:
That's a great suggestion!